@kiva/kv-components/package.json

```diff
@@ -63,6 +63,8 @@
     "embla-carousel": "^4.5.3",
     "focus-trap": "^6.7.2",
     "fuse.js": "^6.6.2",
+    "lodash.groupby": "^4.6.0",
+    "lodash.sortby": "^4.7.0",
```

Comment:
Instead of sortBy, let's make our own util function to do this https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore#_sortby-and-_orderby